test/mocha/helpers.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import {agent} from '@bedrock/https-agent';
+import {httpClient} from '@digitalbazaar/http-client';
```

Comment:
Missing `@digitalbazaar/http-client` in test deps.

Reply:
How did you catch the missing dep? Did you use some library to check? Add it here: https://github.com/digitalbazaar/bedrock-did-resolver-http/pull/3/commits/5e918ab1d3a250ff491ddf658584d5cb314960e5

and thank you.

Reply:
Yeah, I used `depcheck` to check for any missing dependencies.